### PR TITLE
M3/readme & stuff

### PR DESCRIPTION
**Pull request by mauro3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,16 +133,18 @@ dH/dt = dHdt      (1c)
 ```
 The [`diffusion_1D_expl.jl`](scripts/diffusion_1D_expl.jl) code implements an iterative and explicit solution of eq. (1) for an initial Gaussian profile:
 ```md
-H = exp(-(x-lx/2.0)^2)
+H0 = exp(-(x-lx/2.0)^2)
```
Comment (mauro3) [inline README.md:136]: This goes with the slight code-change I did.

Comment (luraess) [reply README.md:136]: This is good with `H0`

Comment (luraess) [reply README.md:136]: From my perspective you can merge @mauro3.

**Comment by luraess** [comment]
From my perspective you can merge @mauro3

**Pull request merged**